src/platforms/dotnet/migration.mdx

```diff
@@ -108,3 +108,183 @@ A new package was added ASP.NET Core gPRC support. With the ability to log the r
 # Release notes
 
 Please refer to [GitHub for the complete release notes](https://github.com/getsentry/sentry-dotnet/releases/tag/3.0.0).
+
+<Note>
```

Comment:
Looking at the version you're updating, I can see you're following that model by adding a note format here, but since we now have two migration guides (updating to version 3.0.5 and updating from SharpRaven), please make both of those heading level 2 so users can see that there are two migration guides on the page.